### ISSUE TITLE
Simplefog missing from Add-on Module Installation Search [BUG]

### ISSUE THREAD

**Issue by kiloforce** [League-of-Foundry-Developers/simplefog#119]
**Module Version:** v0.3.4

**Describe the bug**
When I search for the module in the Module Installation, I am not able to find it in the search.  Fortunately, I can directly install the module using the github module.json url.

**To Reproduce**
1. Load FoundryVTT Admin, and go to "Add-on Modules"
2. Click on "Install Module" and search for "simplefog"

**Expected behavior**
Simplefog should be found in the search.  It was in the past.

**Screenshots**
![image](https://github.com/League-of-Foundry-Developers/simplefog/assets/433058/3f36b407-10f9-4478-a31d-f52c67895c13)

**Browser:**
Tried in both Chrome and Firefox

**Foundry Version:** 
Version 11 Stable - Build 309

**Comment by UranusBytes** [comment]
Thanks for reporting.  As best as I can tell, this is because the module supported version is out of date in the admin console.  I'm at the mercy of others with access to update this.  It's been 2 months since I last asked them; let me try again...

**Comment by r0guebyte** [comment]
Is this also why I can't install it with the Manifest URL onto Version 11 of Foundry?

**Comment by kiloforce** [comment]
Good news, maybe...

I checked again, and it looks like simplefog appears in the index now.  I am using Foundry Version 11 Build 315, if that makes any difference.

![image](https://github.com/League-of-Foundry-Developers/simplefog/assets/433058/f0088cfd-ac7b-4a0f-bf81-93330ffcf3d9)

Only problem is it looks like it is now pointing to https://github.com/mclemente/simplefog

Is that where it should be pointing to now?

**Comment by r0guebyte** [comment]
Seems this has been forked a bunch of times. Not sure there’s an "official" branch. Lol

**Comment by UranusBytes** [comment]
The simplefog from this repo is dead since it couldn't get updated in admin console. I would suggest reporting this bug (if still applicable) here - https://github.com/mclemente/simplefog
